test/Bruno/environments/Local.bru

```diff
@@ -1,10 +1,10 @@
 vars {
-  Environment: dev
+  Environment: at22
```

Comment:
Revert these changes. Though we do need a better method of testing port forwarded endpoints.